jk_script3.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#vytvoreni adresare a souboru v nem
+echo "vytvoreni adresare a souboru"
+echo "============================"
+TARGET_DIR="BeeItScripts"
+TARGET_FILE="example_file1.txt"
+mkdir -p "$TARGET_DIR"
+echo "Domaci ukol cislo 3" > "$TARGET_DIR/$TARGET_FILE"
+echo "Soubor $TARGET_FILE vytvoren v adresari $TARGET_DIR"
+echo "============================"
+#vytvoreni soft a hard linku
+echo "skript vytvori soft a hard link na vyse uvedeny soubor v /tmp"
+ln -s "$(pwd)/$TARGET_DIR/$TARGET_FILE" /tmp/soft_link_example_file1.txt
+ln "$(pwd)/$TARGET_DIR/$TARGET_FILE" /tmp/hard_link_example_file1.txt
+echo "Soft a Hard link vytvoren"
+echo "============================"
+#vypise vsechny instalovane balicky
+echo "===Vypis vsech instalovanych balicku==="
+dpkg --get-selections
```

Comment:
`dpkg` je take moznost pro vylistovani balicku, ale lepsi je pouzivat `apt list --installed`. `dpkg` je sice package manager, ale na rozdil od `apt`, tak neumoznuje instalaci z internetu (nema online/remote repozitare), tzn. ze ten program, ktery clovek chce nainstalovat, tak musi byt stazeny jako `deb` balicek a potom to lze nainstalovat pres `dpkg`